enhancements/installer/single-node-installation-bootstrap-in-place.md

```diff
@@ -166,26 +166,37 @@ logic, the modified script will:
    kubernetes resources, and the bootstrap etcd database snapshot to
    create a new Ignition config for the host.
 
-In order to successfully complete the installation, the user must tell
-`coreos-install` where to write the operating system image on the
-host's local storage. It may be necessary to provide other inputs to
-`coreos-install` at the same time (network addresses, etc.), and we
-have not yet completed the investigation and experimentation work
-required to design a supportable API for the complex details that may
-be needed. Therefore, to unblock progress on the bootstrap-in-place
-feature, we will temporarily treat installing the operating system as
-a manual step. Users running the OpenShift installer by hand will need
-to login to the host via `ssh` and run `coreos-install`, passing the
-appropriate arguments. Users relying on integration with Hive and the
-assisted installer for multi-cluster management will not need to
-perform this step by hand, since the agent driven by the assisted
-installer service runs `coreos-install` already. The installer and
-edge teams will work on this area further during this next release
-cycle, and use another enhancement proposal to describe the
-implementation of an API to replace the manual step.
-
-After the user runs the `coreos-install` and the RHCOS image is
-written to disk, they also need to reboot the host.
+In order to successfully complete the installation, we must know
+where to write the operating system image on the host's local
+storage. Initially this information will be passed as a single value
+in the `install-config.yaml`, using a schema like
+
+```yaml
+coreOSInstallation:
```

Comment:
Can we reserve the right to change the name of this field?

Reply:
Sure, but once we release 4.8 it's there forever 😉

Reply:
Yes, absolutely. I tried to describe that in the text, but if that's not clear we can update the wording. Maybe propose an update on Eran's PR now that this has merged?